dataactvalidator/config/sqlrules/a11_appropriations.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
+    approp.row_number,
+    approp.spending_authority_from_of_cpe,
+    SUM(sf.amount) as total_amount
```

Comment:
Same comment as above re: aliases for aggregate columns. I think doing a global change of `total_amount` to `sf_133_sum` would go a long way towards consistency with other rules w/o being a heavy lift.